torch/_torch_docs.py

```diff
@@ -166,6 +166,19 @@ def merge_dicts(*dicts):
 See :doc:`/notes/randomness` for more information.""",
 }
 
+sparse_support_notes = {
+    "sparse_arg_support": """This operation has support for arguments with :ref:`sparse layouts<sparse-docs>` \
+the layout of all operands will determine the output layout. In the case where attr:`out` is provided it's layout will \
```

Comment:
nit(?): "In the case where attr:`out` is provided it's layout will *be* used without modification."

Reply:
nit: "This operation has support for arguments with :ref:`sparse layouts<sparse-docs>` the layout of all operands will determine the output layout." - Maybe missing a period or sentence separation? Also is it true to say "If all arguments are sparse, the output will be a sparse layout as well. Otherwise the output will be strided."?

Reply:
> Also is it true to say "If all arguments are sparse, the output will be a sparse layout as well. Otherwise the output will be strided."?

That is not the point I was trying to get across with this. The goal is to indicate that for example in some cases the output layout is deduced and providing an additional parameter may cut out that process. Say someone calls `torch.mm(A(csr), B(csr))` on cpu, it succeeds and return `CSR`, then they do the same on GPU, and again success. Now move to addmm: `torch.addmm(self(strided), A(csr), B(csr))` this will work on cpu, but fail on GPU. We don't have support for that call yet. Basically I am trying to convey a warning that the success of the `mm` call is not a reliable indication that the similar `addmm` call will succeed, also that although it fails on some backends, passing `self` with different layout will affect the layout support equation. 


To answer your question: If all arguments are sparse, the output will be sparse this is true. If not, the output will not necessarily be strided the `addmm` example above would have strided return (if successful), however rotating the layouts of the arguments, for example `torch.addmm(self(csr), A(csr), B(strided))` will attempt to return `csr` output, however I believe this call will error on all backends.

Reply:
Looks like maybe we need separate documentation for addmm?